ListApp/Tests/ListApp.Api.Tests/ItemControllerTests.cs

```diff
@@ -62,60 +94,69 @@ public void SetUp()
             var receivedResponse = await _itemsController.GetAsync(Guid.NewGuid());
             var responseMessage = await receivedResponse.ExecuteAsync(CancellationToken.None);
 
+            Assert.DoesNotThrowAsync(() => _itemsRepository.Received(1).GetAsync(Arg.Any<Guid>()));
```

Comment:
This DoesNotThrowAsync call is pointless, the Received throws nice enough exception that fails the test as well.